ownCloud/Client/Viewer/PDF/PDFViewerViewController.swift

```diff
@@ -149,15 +173,18 @@ class PDFViewerViewController: DisplayViewController, DisplayExtension {
 				pdfView.delegate = self
 				containerView.addArrangedSubview(pdfView)
 
+				pageCountButton.translatesAutoresizingMaskIntoConstraints = false
+				pageCountButton.accessibilityLabel = "Go to page".localized
+				pageCountButton.addTarget(self, action: #selector(goToPage), for: .touchDown)
 				pageCountContainerView.translatesAutoresizingMaskIntoConstraints = false
-				pageCountContainerView.addSubview(pageCountLabel)
+				pageCountContainerView.addSubview(pageCountButton)
 
-				pageCountLabel._setupPdfInfoLabel()
+				pageCountButton.centerXAnchor.constraint(equalTo: pageCountContainerView.centerXAnchor).isActive = true
+				pageCountButton.centerYAnchor.constraint(equalTo: pageCountContainerView.centerYAnchor).isActive = true
+				pageCountButton.widthAnchor.constraint(equalTo: pageCountContainerView.widthAnchor, multiplier: 0.25).isActive = true
+				pageCountButton.heightAnchor.constraint(equalTo: pageCountContainerView.heightAnchor, multiplier: 0.5).isActive = true
+				pageCountButton.heightAnchor.constraint(lessThanOrEqualToConstant: 22.0).isActive = true
```

Comment:
Looks fine, but please consider adding constraints using
```swift
NSLayoutConstraint.activate([
    // … constraints here …
])
```
as it removes some noise (`.isActive = true` in every line).